firebase-sessions/src/test/kotlin/com/google/firebase/sessions/testing/FakeProcessDetails.kt

```diff
@@ -22,4 +22,6 @@ import com.google.firebase.sessions.ProcessDetails
 class FakeProcessDetails(
   override val isDefaultProcess: Boolean = true,
```

Comment:
If we have the process name and default process name in the interface, do we need this isDefaultProcess anymore?

Reply:
I'll remove.